librz/main/rz-diff.c

```diff
@@ -183,6 +218,11 @@ static void rz_diff_show_help(bool usage_only) {
 		"              sections   | compares sections found in the files\n"
 		"              strings    | compares strings found in the files\n"
 		"              symbols    | compares symbols found in the files\n"
+		"  palette colors can be changed by adding the following lines\n"
```

Comment:
Could you set them in the default theme at least? And maybe move this piece of the documentation in the `rz-diff` man page? It doesn't really belong in the `-h` output, I think.

Reply:
the are already